kong/pdk/request.lua

```diff
@@ -39,7 +39,7 @@ local function new(self)
   local MIN_HEADERS            = 1
   local MAX_HEADERS_DEFAULT    = 100
   local MAX_HEADERS            = 1000
-  local MIN_QUERY_ARGS         = 1
+  local MIN_QUERY_ARGS         = 0
```

Comment:
This change isn't documented nor tested. What's the intent behind this?

Reply:
This is documented [here](https://github.com/openresty/lua-nginx-module#ngxreqget_uri_args).
When set to 0 it means no limit to the number.